src/main/java/com/microsoft/jenkins/azuread/AzureAdUser.java

```diff
@@ -68,6 +69,14 @@ public static AzureAdUser createFromJwt(JwtClaims claims) throws MalformedClaimE
         if (StringUtils.isEmpty(user.uniqueName)) {
             user.uniqueName = (String) claims.getClaimValue("preferred_username");
         }
+        for (String suffix : unwantedUsernameSuffixes.split(",")) {
```

Comment:
NullPointer issue?

Reply:
Shouldn't be, `unwantedUsernameSuffixes` is an empty string (default value) or string with a content (unmarshalling empty node returns `""` as well).

Reply:
Writing this test seems problematic.
Well, I double checked used methods, in the end, `reader.getValue();` turns into:
```java
    public String getValue() {
        return current.value == null ? EMPTY_STRING : current.value;
    }
```

Reply:
Thanks for contribute. I need do some test.

Reply:
Hi @xuzhang3, any update?